### DISCUSSION TITLE
Cellular Data

### DISCUSSION THREAD

**Discussion by mohoyek** [B-andi-T/Openconnect-installer-Full-Management-Menu#2] [Q&A]
hi 
cellular data dosent work
operation MTN , MCI

**Comment by B-andi-T** [comment]
> hi cellular data dosent work operation MTN , MCI

If your mobile operator has blocked access to ocserv, you can use SSH tunnel. read this : 
https://github.com/B-andi-T/Openconnect-installer-Full-Management-Menu#do-you-live-in-a-country-where-ocserv-is-blocked
